packages/react-table/src/components/Table/utils/decorators/collapsible.tsx

```diff
@@ -19,23 +19,38 @@ export const collapsible: IFormatter = (
     property
   };
 
+  const rowId = rowIndex !== undefined ? rowIndex : -1;
+
+  const customProps = {
+    ...(rowId !== -1
+      ? {
+          isOpen: rowData?.isOpen,
+          'aria-labelledby': `${rowLabeledBy}${rowId} ${expandId}${rowId}`
+        }
+      : {
+          isOpen: allRowsExpanded,
+          'aria-label': 'Expand all rows'
```

Comment:
Can this area label be customized?

Reply:
the expandAllRowsButtonAriaLabel would have to be set and passed the same way allRowsExpanded is passed.

Reply:
this is similar to how it is for the select all, i am looking into how to get it customizable.. looking like i'll have to add a property for the table which would need to get passed through